ios-shoppe-demo/ios-shoppe-demo/ios-shoppe-demo/APIService.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  APIService.swift
+//  ios-shoppe-demo
+//
+//  Created on 3/4/20.
+//  Copyright © 2020 FullStory All rights reserved.
+//
+
+import Foundation
+
+class APIService {
+    static var sharedInstance = APIService()
+
+    var baseURL: String = "http://reactshoppe.s3-website-us-east-1.amazonaws.com/data/products.json"
+    var shopItems: [[String : Any]] = [] // TODO: Change this to be an array of class ShopItem
```

Comment:
I used "product" in stead of "shopItem" in my code, should we use the same names? If so, what's a good generic name? for this

Reply:
lets chat on slack on it @van-fs liked the idea of ShoppeItem when I posed the question in our channel